File: combine_script.py
#!/usr/bin/env python3
import subprocess
import sys

# Get the original file from git
result = subprocess.run(['git', 'show', 'HEAD:public/script.js'], 
                       capture_output=True, text=True, cwd='.')

if result.returncode != 0:
    print(f"Error: {result.stderr}")
    sys.exit(1)

lines = result.stdout.split('\n')
print(f"Total lines in original: {len(lines)}")

# Extract the two parts
part1 = lines[:1779]  # Lines 1-1779
part2 = lines[2252:]  # Lines 2253 onwards

print(f"Part 1 lines: {len(part1)}")
print(f"Part 2 lines: {len(part2)}")

# Combine
combined = part1 + part2

# Write to file
with open('public/script.js', 'w', encoding='utf-8') as f:
    f.write('\n'.join(combined))

print(f"File recreated with {len(combined)} lines")

File: combine_script.js
const fs = require('fs');
const { execSync } = require('child_process');

try {
  // Get original file from git
  const original = execSync('git show HEAD:public/script.js', { encoding: 'utf-8' });
  const lines = original.split('\n');
  
  console.log(`Total lines: ${lines.length}`);
  
  // Extract parts
  const part1 = lines.slice(0, 1779);
  const part2 = lines.slice(2252);
  
  console.log(`Part 1: ${part1.length} lines`);
  console.log(`Part 2: ${part2.length} lines`);
  
  // Combine
  const combined = [...part1, ...part2];
  
  // Write
  fs.writeFileSync('public/script.js', combined.join('\n'), 'utf-8');
  
  console.log(`File recreated: ${combined.length} lines`);
} catch (error) {
  console.error('Error:', error.message);
  process.exit(1);
}
